spec/lib/api/v3/projects/project_representer_spec.rb

```diff
@@ -65,6 +65,33 @@
         expect(subject).to have_json_path('_links/self/title')
       end
 
+      describe 'create work packages' do
+        let(:user) do
+          FactoryGirl.build(:user, member_in_project: project, member_through_role: role)
+        end
+        let(:project) { FactoryGirl.create(:project) }
+        let(:representer) { described_class.new(project, current_user: user) }
```

Comment:
I'd push those `let` statements up to the root of the spec, check if on the happy path all links are displayed and then only have a single context for the unhappy path where the permissions are not sufficient.